styles/web/_vars.scss

```diff
@@ -750,1222 +250,975 @@ $sprk-btn-tertiary-border: $sprk-btn-border-width $sprk-btn-border-style
+/// The border color of the huge checkbox input when checked and hovered.
+$sprk-checkbox-huge-container-checked-hover-border-color: $sprk-black !default;
+/// The max-width of the huge checkbox input.
+$sprk-checkbox-input-huge-max-width: 500px !default;
+/// The height of the custom checkbox input.
+$sprk-checkbox-custom-input-height: 1rem !default;
+/// The width of the custom checkbox input.
+$sprk-checkbox-custom-input-width: 1rem !default;
+/// The border of the custom checkbox input.
+$sprk-checkbox-custom-input-border: 2px solid $sprk-black-tint-50 !default;
+/// The border of the custom checkbox input when focused.
+$sprk-checkbox-custom-input-border-focus: 2px solid $sprk-purple !default;
+/// The border of the custom checkbox input when it is checked and focused.
+$sprk-checkbox-custom-input-border-focus-checked: 2px solid $sprk-black !default;
+/// The background of the custom checkbox input when focused.
+$sprk-checkbox-custom-input-background-color-focus: $sprk-gray !default;
+/// The background color of the custom checkbox input when hovered.
+$sprk-checkbox-custom-input-background-color-hover: $sprk-gray !default;
+/// The background color of the custom checkbox input when checked.
+$sprk-checkbox-custom-input-background-color-checked: $sprk-purple !default;
+/// The background color of the custom checkbox input when disabled.
+$sprk-checkbox-custom-input-background-color-disabled: transparent !default;
+/// The border color of the custom checkbox input when checked.
+$sprk-checkbox-custom-input-border-color-checked: $sprk-purple !default;
+/// The border color of the custom checkbox input when hovered.
+$sprk-checkbox-custom-input-border-color-hover: $sprk-black-tint-70 !default;
+/// The border color of the custom checkbox input when disabled.
+$sprk-checkbox-custom-input-border-color-disabled: $sprk-black-tint-50 !default;
+/// The cursor for the custom checkbox input when disabled..
+$sprk-checkbox-custom-input-cursor-disabled: default !default;
+/// The border radius of the custom checkbox input.
+$sprk-checkbox-custom-input-border-radius: 3px !default;
+/// The top positioning of the custom checkbox input.
+$sprk-checkbox-custom-input-top: 0 !default;
+/// The left positioning of the custom checkbox input.
+$sprk-checkbox-custom-input-left: -22px !default;
+/// The height of the custom checkbox element of the huge checkbox input.
+$sprk-checkbox-huge-custom-input-height: 1.5rem !default;
+/// The width of the custom checkbox element of the huge checkbox input.
+$sprk-checkbox-huge-custom-input-width: 1.5rem !default;
+/// The border color of the custom checkbox element of the huge checkbox input when hovered.
+$sprk-checkbox-huge-custom-input-hover-border-color: $sprk-black !default;
+/// The top value of the custom checkbox element of the huge checkbox input..
+$sprk-checkbox-huge-custom-input-top: $sprk-space-m !default;
+/// The left value of the custom checkbox element of the huge checkbox input..
```

Comment:
This needs to be changed in the input.json file. Because that file is the source. Remove there then regenerate the sass file buy starting any one of the storybooks, doc site or by running the build command in styles.

Reply:
Fixed a bunch of the source files in this PR https://github.com/sparkdesignsystem/spark-design-system/pull/3663/files

Reply:
https://github.com/sparkdesignsystem/spark-design-system/pull/3663/files#diff-825c7ae6fbfba993b3068a4ab8bcfa17cc768e690c46658a323862bbc80eca65